sklearn/feature_extraction/hashing.py

```diff
@@ -40,11 +40,11 @@ class FeatureHasher(BaseEstimator, TransformerMixin):
         The number of features (columns) in the output matrices. Small numbers
         of features are likely to cause hash collisions, but large numbers
         will cause larger coefficient dimensions in linear learners.
-    dtype : numpy type, optional
+    dtype : numpy type, optional, default np.float64
         The type of feature values. Passed to scipy.sparse matrix constructors
         as the dtype argument. Do not set this to bool, np.boolean or any
         unsigned integer type.
-    input_type : string, optional
+    input_type : string, optional, default "dict"
```

Comment:
I know we're inconsistent about how we indicate defaults. Perhaps because this case is not semantically transparent, it is not helpful to say the default here when it is immediately shown in the description...